In [ ]:
1. Write a Python program to take two numbers as input from the user and display their sum.

1. Write a Python program to take two numbers as input from the user and display their sum.

In [ ]:
a = int(input("Enter a first number="))
b = int(input("Enter a second number="))
sum = a+b
print("addition of two numbers=", sum)

Enter a first number=10
Enter a second number=20
addition of two numbers= 30


2.  **Calculate and print the midpoint of two numbers entered by the user.**
 **bold text**

In [ ]:
a = int(input("Enter a first number="))
b = int(input("Enter a second number="))
midpoint = (a+b)/2
print("midpoint of two numbers=" , midpoint)

Enter a first number=10
Enter a second number=20
midpoint of two numbers= 15.0


**Write a Python program to calculate the area of a triangle using the formula: area = 0.5 * base * height.**bold text**

In [ ]:
base = int(input("enter a base number="))
height = int(input("enter a base number="))
area = 0.5*base*height
print("area of triangle=", area)

enter a base number=50
enter a base number=25
area of triangle= 625.0


**Take three numbers as input from the user and display their **product**

In [ ]:
Num1 = int(input("enter a first number="))
Num2 = int(input("enter a second number="))
Num3 = int(input("enter a third number="))
Sum = (Num1*Num2*Num3)
print("product of three number=", Sum)


enter a first number=45
enter a second number=98
enter a third number=75
product of three number= 330750


. Write a Python program to take two numbers as input from the user and display their sum.

[ ]
a = int(input("Enter a first number="))
b = int(input("Enter a second number="))
sum = a+b
print("addition of two numbers=", sum)


Enter a first number=54
Enter a second number=76
addition of two numbers= 130

Calculate and print the midpoint of two numbers entered by the user.

[ ]
a = int(input("Enter a first number="))
b = int(input("Enter a second number="))
midpoint = (a+b)/2
print("midpoint of two numbers=" , midpoint)


Enter a first number=87
Enter a second number=545
midpoint of two numbers= 316.0


Write a Python program to calculate the area of a triangle using the formula: area = 0.5 * base * height.

[ ]
base = int(input("enter a base number="))
height = int(input("enter a base number="))
area = 0.5*base*height
print("area of triangle=", area)


enter a base number=78
enter a base number=43
area of triangle= 1677.0


Take three numbers as input from the user and display their product.

[ ]





enter a first number=45
enter a second number=98
enter a third number=75
product of three number= 330750



Take a number as input and display its square and cube.


. Write a Python program to take two numbers as input from the user and display their sum.

[ ]
a = int(input("Enter a first number="))
b = int(input("Enter a second number="))
sum = a+b
print("addition of two numbers=", sum)


Enter a first number=54
Enter a second number=76
addition of two numbers= 130

Calculate and print the midpoint of two numbers entered by the user.

[ ]
a = int(input("Enter a first number="))
b = int(input("Enter a second number="))
midpoint = (a+b)/2
print("midpoint of two numbers=" , midpoint)


Enter a first number=87
Enter a second number=545
midpoint of two numbers= 316.0


Write a Python program to calculate the area of a triangle using the formula: area = 0.5 * base * height.

[ ]
base = int(input("enter a base number="))
height = int(input("enter a base number="))
area = 0.5*base*height
print("area of triangle=", area)


enter a base number=78
enter a base number=43
area of triangle= 1677.0


Take three numbers as input from the user and display their product.

[ ]





enter a first number=45
enter a second number=98
enter a third number=75
product of three number= 330750



Take a number as input and display its square and cube.


In [ ]:
Num = int(input("enter a number="))

print("sqaure of a number=", Num**2)
print("cube of a number=", Num**3)


enter a number=25
sqaure of a number= 625
cube of a number= 15625


6.  Take two numbers as input and display their quotient and remainder.


In [ ]:
Num1 = int(input("enter a first number ="))
Num2 = int(input("enter a second number="))

print("quotient of number=", Num1 // Num2)
print("remainder of number=", Num1 % Num2)

enter a first number =48
enter a second number=78
quotient of number= 0
remainder of number= 48


Take three numbers as input and display their average.


In [ ]:
Num1 = int(input("enter a first number="))
Num2 = int(input("enter a second number="))
Num3 = int(input("enter a third number="))
print("average of three numbers=", (Num1+Num2+Num3)/3)

enter a first number=95
enter a second number=43
enter a third number=80
average of three numbers= 72.66666666666667


8.  Write a Python program to calculate the area of a rectangle using the formula: area = length * width.


In [ ]:
length = int(input("enter a length number="))
width = int(input("enter a width number="))

print("area of rectangle=", (length*width))

enter a length number=67
enter a width number=87
area of rectangle= 5829


Take five numbers as input representing marks obtained and total marks.
Display the percentage.

In [ ]:
maths = int(input("enter a maths number="))
science = int(input("enter a science number="))
marathi = int(input("enter a marathi number="))
hindi = int(input("enter a hindi number="))
sanskrit = int(input("enter a sanskrit number="))

print("percentage of five numbers=", ((maths+science+marathi+hindi+sanskrit)/500)*100)

enter a maths number=90
enter a science number=96
enter a marathi number=76
enter a hindi number=85
enter a sanskrit number=80
percentage of five numbers= 85.39999999999999


10. Write a Python program to swap two numbers entered by the user without using a third variable.


In [ ]:
# with easy method
a = int(input("enter a first number="))
b = int(input("enter a second number="))
a,b = b,a
print("a =",a)
print("b =",b)

enter a first number=56
enter a second number=12
a = 12
b = 56


In [ ]:
# with arithematic method
a = int(input("enter a first number="))
b = int(input("enter a second number="))

a = a+b
b = a-b
a = a-b
print("a =",a)
print("b =",b)


enter a first number=23
enter a second number=78
a = 78
b = 23


Take a number as input and check whether it is even or odd.

In [ ]:
Num = int(input("enter a number="))

if Num % 2 == 0:
  print("even number")
else:
  print("odd number")

enter a number=453
odd number


Write a Python program to convert temperature from Celsius to Fahrenheit using the formula: F = (C × 9/5) + 32

In [ ]:
C = int(input("enter tempreture in celsius="))
F = (C*9/5)+32
print("convert tempreture from celsius to fahrenheit=", F )

enter tempreture in celsius=35
convert tempreture from celsius to fahrenheit= 95.0


Write a Python program to convert temperature from Fahrenheit to Celsius using the formula: C = (F - 32) × 5/9

In [ ]:
F = int(input("enter tempreture in celsius="))
C = (F -32)* 5/9
print("convert tempreture from fahrenheit to celsius=", C )

enter tempreture in celsius=213
convert tempreture from fahrenheit to celsius= 100.55555555555556


Take two numbers as input and find their greatest common divisor (GCD)

In [ ]:
import math
A = int(input("enter a first number="))
B = int(input("enter a second number="))

gcd = math.gcd(A, B)
print(f"gcd of {A} and {B} is {gcd}")


enter a first number=678
enter a second number=346
gcd of 678 and 346 is 2


Write a Python program to calculate the simple interest using the formula: SI = (P × R × T) / 100, where P is the principal amount, R is the rate of interest, and T is the time in years

In [ ]:
P = int(input("enter a principal amount="))
R = int(input("enter a rate of interest="))
T = int(input("enter a time in years="))
SI = (P*R*T)/100
print("The simple interest=", SI)


enter a principal amount=500000
enter a rate of interest=12
enter a time in years=7
The simple interest= 420000.0


Write a Python program to check whether a number is positive, negative, or zero.


In [ ]:
num = float(input("Enter a number: "))

if num > 0:
    print(num, "is Positive")
elif num < 0:
    print(num, "is Negative")
else:
    print("The number is Zero")

Enter a number: 1
1.0 is Positive


**17.Write a Python program to find the largest among three numbers entered by the user.**


In [ ]:
a = int(input("Enter first number: "))
b = int(input("Enter second number: "))
c = int(input("Enter third number: "))

print("The largest number is:", max(a, b, c))

Enter first number: 55
Enter second number: 45
Enter third number: 65
The largest number is: 65


Take a number as input and check whether it is a multiple of 5.**

In [ ]:
num = int(input("Enter number:"))
if num % 5 == 0:
    print(num, "is a multiple of 5")
else:
    print(num, "is not a multiple of 5")


Enter number:55
55 is a multiple of 5


**19.Write a Python program to calculate the perimeter of a circle using the formula: perimeter = 2 * π * radius.**


In [ ]:
import math

radius = float(input("Enter the radius of the circle: "))

perimeter = 2 * math.pi * radius

print("Perimeter of the circle is:", perimeter)

Enter the radius of the circle: 7
Perimeter of the circle is: 43.982297150257104


Write a Python program to check whether a number is divisible by both 3 and 5.**

In [ ]:
num = int(input("Enter number:"))
if num%3==0 and num%5==0:
  print(num,"is divisible by both 3 and 5")
else:
  print(num,"is Not divisible by both 3 and 5")


Enter number:15
15 is divisible by both 3 and 5
